wandb/sdk/launch/runner/kubernetes.py

```diff
@@ -0,0 +1,222 @@
+import datetime
+import os
+import subprocess
+import time
+from typing import Any, Dict, List, Optional
+
+if False:
+    import kubernetes   # type: ignore
+from six.moves import shlex_quote
+import wandb
+import wandb.docker as docker
+from wandb.errors import LaunchError
+from wandb.util import get_module, load_json_yaml_dict
+import yaml
+
+from .abstract import AbstractRun, AbstractRunner, Status
+from .._project_spec import LaunchProject
+from ..docker import (
+    construct_local_image_uri,
+    generate_docker_image,
+    pull_docker_image,
+    validate_docker_installation,
+)
+from ..utils import (
+    PROJECT_DOCKER_ARGS,
+    PROJECT_SYNCHRONOUS,
+)
+
+TIMEOUT = 5
+
+class KubernetesSubmittedRun(AbstractRun):
+    from kubernetes.client.api.batch_v1_api import BatchV1Api
```

Comment:
do we need to add the k8s package to some requirements.txt?

Reply:
in general we don't want to require that all wandb users install k8s — this call should be wrapped in `use_module`, good catch